### PR TITLE
feat: auth with token

### PR DESCRIPTION
**Pull request by xyc**
### What issues does this PR fix or reference?
@W-9163648@

### CODE REVIEW

File: packages/salesforcedx-vscode-core/src/commands/auth/forceAuthWebLogin.ts [hunk 1]
```diff
@@ -126,101 +117,6 @@ export class ForceAuthWebLoginDemoModeExecutor extends ForceAuthDemoModeExecutor
   }
 }
 
-export class OrgTypeItem implements vscode.QuickPickItem {
```
Comment (xyc) [inline packages/salesforcedx-vscode-core/src/commands/auth/forceAuthWebLogin.ts:129]: These have been moved to `authParamsGatherer.ts`.

File: packages/salesforcedx-vscode-core/package.nls.ja.json [hunk 2]
```diff
@@ -2,6 +2,7 @@
   "running_tasks_title_text": "実行中のタスク",
   "force_auth_web_login_authorize_dev_hub_text": "SFDX: Dev Hub を認証",
   "force_auth_web_login_authorize_org_text": "SFDX: 組織を認証",
+  "force_auth_access_token_authorize_org_text": "SFDX: Authorize an Org using Session ID",
```
Comment (xyc) [inline packages/salesforcedx-vscode-core/package.nls.ja.json:5]: @sbudhirajadoc please take a look

File: packages/salesforcedx-vscode-core/src/messages/i18n.ja.ts [hunk 3]
```diff
@@ -52,6 +52,11 @@ export const messages = {
     '組織のエイリアスを入力またはデフォルトのエイリアスを使用',
   parameter_gatherer_enter_custom_url:
     'カスタムのログイン URL を入力またはデフォルトの URL を使用',
+  parameter_gatherer_enter_instance_url: 'Enter Instance URL',
```
Comment (xyc) [inline packages/salesforcedx-vscode-core/src/messages/i18n.ja.ts:55]: @sbudhirajadoc please take a look

Comment (sbudhirajadoc) [reply packages/salesforcedx-vscode-core/src/messages/i18n.ja.ts:55]: looks good!

**Review by rcoringrato-sfdc — APPROVED** [review]

**Pull request merged**